### PR TITLE
PLAT-463: A.Foo -> B1.Bar, B2.Bar, B3.Bar

### PR DESCRIPTION
**Pull request by ElSacramento**
**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: ledger-core/virtual/integration/method_test.go [hunk 1]
```diff
@@ -532,3 +532,151 @@ func TestVirtual_CallContractFromContract_Unordered(t *testing.T) {
...
+		&executionupdate.ContractExecutionStateUpdate{
+			Type:     executionupdate.OutgoingCall,
+			Error:    nil,
+			Outgoing: builder.CallMethod(objectB1Global, classB, "Bar", []byte{}),
+		},
+	)
+
+	objectAExecutionMock.AddContinue(
+		func(result []byte) {
+			t.Log("ExecutionContinue [A.Foo]")
```
Comment (ruz) [inline ledger-core/virtual/integration/method_test.go:583]: I don't see result checked.

File: ledger-core/virtual/integration/method_test.go [hunk 2]
```diff
@@ -532,3 +532,151 @@ func TestVirtual_CallContractFromContract_Unordered(t *testing.T) {
...
+		},
+		&executionupdate.ContractExecutionStateUpdate{
+			Type:     executionupdate.OutgoingCall,
+			Error:    nil,
+			Outgoing: builder.CallMethod(objectB2Global, classB, "Bar", []byte{}),
```
Comment (ruz) [inline ledger-core/virtual/integration/method_test.go:588]: arguments are empty so are not checked

File: ledger-core/virtual/integration/method_test.go [hunk 3]
```diff
@@ -532,3 +532,151 @@ func TestVirtual_CallContractFromContract_Unordered(t *testing.T) {
...
+		},
+		&executionupdate.ContractExecutionStateUpdate{
+			Type:   executionupdate.Done,
+			Result: requestresult.New([]byte("finish B2.Bar"), objectB2Global),
+		},
+	)
+
+	runnerMock.AddExecutionMock(objectB3Global.String()).AddStart(
+		func(ctx execution.Context) {
+			t.Log("ExecutionStart [B3.Bar]")
```
Comment (ruz) [inline ledger-core/virtual/integration/method_test.go:634]: nothing is checked here except fact of the call

File: ledger-core/virtual/integration/method_test.go [hunk 4]
```diff
@@ -532,3 +532,151 @@ func TestVirtual_CallContractFromContract_Unordered(t *testing.T) {
...
+		},
+	)
+
+	runnerMock.AddExecutionClassify(objectAGlobal.String(), flags, nil)
+	runnerMock.AddExecutionClassify(objectB1Global.String(), flags, nil)
+	runnerMock.AddExecutionClassify(objectB2Global.String(), flags, nil)
+	runnerMock.AddExecutionClassify(objectB3Global.String(), flags, nil)
+
+	typedChecker := server.PublisherMock.SetTypedChecker(ctx, mc, server)
+	typedChecker.VCallRequest.SetResend(true).ExpectedCount(3)
```
Comment (ruz) [inline ledger-core/virtual/integration/method_test.go:648]: no fields of VCallRequests are checked, I many to be checked, especially Sequence.

File: ledger-core/virtual/integration/method_test.go (hunk 1 above)
Comment (ElSacramento) [reply ledger-core/virtual/integration/method_test.go]: done

File: ledger-core/virtual/integration/method_test.go (hunk 2 above)
Comment (ElSacramento) [reply ledger-core/virtual/integration/method_test.go]: done

File: ledger-core/virtual/integration/method_test.go (hunk 3 above)
Comment (ElSacramento) [reply ledger-core/virtual/integration/method_test.go]: done

File: ledger-core/virtual/integration/method_test.go (hunk 4 above)
Comment (ElSacramento) [reply ledger-core/virtual/integration/method_test.go]: done

**Pull request merged**